sectors/wellness.md

```diff
@@ -1 +1,5 @@
-# 😌 Wellness
\ No newline at end of file
+# 😌 Wellness
+
+## Courses
+
+*[Wellbeing course](https://www.coursera.org/learn/the-science-of-well-being?utm_source=gg&utm_medium=sem&utm_content=09-ScienceofWellBeing-ROW&campaignid=9722859275&adgroupid=96434822781&device=c&keyword=science%20of%20well%20being&matchtype=b&network=g&devicemodel=&adpostion=&creativeid=428916120685&hide_mobile_promo&gclid=Cj0KCQjwxNT8BRD9ARIsAJ8S5xZFNcbGXO-dUAlBGdmAfBLLaual55o9bVYv2AaykFni43sGsUh25rMaAi8zEALw_wcB) - Free Coursera course on wellbeing
```

Comment:
Looks great! To make the raw markdown look a little cleaner, everything after `the-science-of-well-being` in the link can be removed as it is just analytics data. Once that's cleaned I'll merge it! :tada: 👍🏻